lib/backend/dynamo/dynamodbbk.go

```diff
@@ -97,11 +100,20 @@ type Config struct {
 // CheckAndSetDefaults is a helper returns an error if the supplied configuration
 // is not enough to connect to DynamoDB
 func (cfg *Config) CheckAndSetDefaults() error {
+
 	// Table name is required.
 	if cfg.TableName == "" {
 		return trace.BadParameter("DynamoDB: table_name is not specified")
 	}
 
+	// Check the TableName is not being used within the audit uri list
+	dynamoDBURI := regexp.MustCompile(`((?i)dynamodb)://` + cfg.TableName)
```

Comment:
This variable does not hold a URI, so I don't think the name is appropriate.

Reply:
@zmb3 renamed here and in the firestore check